### DISCUSSION TITLE
Be able to show blocking confirmation modal

### DISCUSSION THREAD

**Discussion by alexleboucher** [directus/directus#15362] [Feature Requests]
Hi !
Could you add a way to show a blocking confirmation modal that asks the user to confirm he wants do an action but depending on condition.
By example, if an admin creates a user with the same name than an existing user, show a confirmation modal :
`A user with this name already exists, are you sure you want to create this one ?`
By example in the filter hook code ?

Thank you !

**Comment by rijkvanzanten** [comment]
In this particular case, you could add a Unique constraint to the User's name 🙂 

That being said, for more custom use cases, this is where https://github.com/directus/directus/pull/15283 comes in. That will allow you to error out in a custom Flow and return any message you'd like for more in-depth validation pieces

**Reply by alexleboucher** [reply]
Hey, 
Yes but I don't want to have an Unique constraint or throw an error. In my case, 2 users can have the same name but it's rare so I'd like a way to show a confirmation modal to be sure it's what the user wants. 
I use the example with name but it could be something else.

**Reply by rijkvanzanten** [reply]
Ohh I see. There's currently no real way to have the user confirm that change 🤔 If you were to use a flow to cancel it out, the next attempt would again cancel out, so you'd end up with the same situation as a unique constraint

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
